src/client/Microsoft.Identity.Client/MsalServiceExceptionFactory.cs

```diff
@@ -93,6 +93,26 @@ internal class MsalServiceExceptionFactory
             return ex;
         }
 
+        internal static MsalServiceException FromGeneralHttpResponse(
```

Comment:
Consider renaming the other method "FromEstsTokenResponse"

Reply:
Renamed.